### ISSUE TITLE
BWA stuck on this sequence?

### ISSUE THREAD

**Issue by harveyche** [lh3/bwa#261]
BWA version 0.7.12
Indexed Genome GRCh38

bwa mem -t 1 BWA_GrCh38 input.fq > output.sam

when I try to map the following sequence to GRCh38 genome. BWA stuck forever and do not stop. Can anyone reproduce it?

input.fq
@V300037498L4C001R0040237633/2
GGCTCTGCAGACAGAGCTGCAATGAAGACAGACACCGTTCCCTGCCCCCCCCCCCCCCCCCCCACCCCCCCCCCCCCTCCCCCCTCCCCTCCCCCCGCGC
+
FFFEFFFFFF9FFFEFFCFFFFFFFFFEBFFFFFFEB;13;;*5.2;CDA7:8@AB=,?@86C)4B*/:.*B%+B73.D4CA)5&7>8709;1,4/<'(,

**Comment by yanlinlin82** [comment]
No, I tried both bwa 0.7.12-r1039 and 0.7.17-r1188 but haven't reproduced it. Maybe you can try either recompile bwa from source or reindex the reference genome.

**Comment by harveyche** [comment]
If I remove the low quality sequence at the 3' this read can be mapped successfully, but I got many low quality reads in my samples.

I have also re-compiled and tried latest version of the BWA and reindex reference but no good.
The genome I use is from:
ftp://ftp.ensembl.org/pub/release-98/fasta/homo_sapiens//dna/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz

Which reference genome do you use?

**Comment by yanlinlin82** [comment]
I tried the reference genome file you provided, and it goes well without any error or blocking.
